app/views/site/coc.html.haml

```diff
@@ -11,34 +14,172 @@
     Contact [help@indyhackers.org](mailto:help@indyhackers.org).
 
     If you want to chat about something/report a violation you can contact any of the current board members:
```

Comment:
Agreed... I put a link to our info at the top, but moved them below the coc content